### ISSUE TITLE
Bug with const scope

### ISSUE THREAD

**Issue by monolithed** [6to5/6to5#331]
```js
var foo = ( ) => {
	var bar = 0;

	return bar *= 2;
};

foo();

const bar = 0;
```

```
Running "6to5:build" (6to5) task
Warning: trunk/stage/index.js: Line 71: bar is read-only
  69   | 	test () {
  70   | 		var foo = ( ) => {
> 71   | 			var bar = 0;
       |       ^
  72   | 
  73   | 			return bar *= 2;
  74   | 		}; Use --force to continue.
```

**Comment by cvrebert** [comment]
Still broken as of v3.0.13

**Comment by sebmck** [comment]
@cvrebert I know, which is why it's still opened :stuck_out_tongue_winking_eye:

**Comment by sebmck** [comment]
Finally got around to this... Fixed as of 3.3.5, thanks!
